go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/AlecAivazis/survey/v2 v2.2.9
 	github.com/andygrunwald/go-jira v1.13.0
+	github.com/coreos/go-semver v0.3.0 // indirect
```

Comment:
This is a direct dependency, `// indirect` should be removed.

Reply:
yup you're right. a `go mod tidy` fixed this up.